mmv1/third_party/terraform/tests/resource_cloudfunctions_function_test.go.erb

```diff
@@ -146,6 +146,8 @@ func TestAccCloudFunctionsFunction_basic(t *testing.T) {
 						"available_memory_mb", "128"),
 					resource.TestCheckResourceAttr(funcResourceName,
 						"max_instances", "10"),
+          resource.TestCheckResourceAttr(funcResourceName,
```

Comment:
Why is this expected? `testAccCloudFunctionsFunction_basic` doesn't have the value set.

Reply:
Ah, this should be updated. I'll do that.